src/main/java/study/step3/model/car/Cars.java

```diff
@@ -0,0 +1,48 @@
+package study.step3.model.car;
+
+import study.step3.view.ResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Cars {
+
+    private List<Car> racingGameCars;
+
+    public Cars(int carNumber) {
+        this.racingGameCars = new ArrayList<>();
+        addCars(carNumber);
+    }
```

Comment:
```suggestion
    prviate Cars(List<Car> cars) {
        this.racingGameCars = cars;
    }
    
    public static of Cars(List<Car> cars) {
        return new Cars(cars);
    }
    
    public static of Cars(int carNumber) {
        return Stream.generate(Car::new)
                .limit(carNumber)
                .collect(collectingAndThen(toList(), Cars::of));
    }
```
- 이렇게 정적 팩토리 메소드 패턴을 이용해보면 어떨까요?
-  `다형성`을 생각해보면서 작성해주면 어떨까요?

객체지향의 `다형성`이라는 키워드에 대해 한 번 검색해보세요!